src/modules/media/controllers/media.controller.ts

```diff
@@ -26,6 +26,15 @@ export class MediaController {
 		return url;
 	}
 
+	@Get('thumbnail-ticket')
+	public async getThumbnailUrl(
+		@Headers('referer') referer: string,
+		@Query() thumbnailQuery: ThumbnailQueryDto
```

Comment:
Why don't you use a route param for the representationId?
```suggestion
	@Get('thumbnail-ticket/:representationId')
	public async getThumbnailUrl(
		@Headers('referer') referer: string,
		@Param() representationId: string
```